modules/manage/partials/rbac-delete-role.adoc

```diff
@@ -0,0 +1,11 @@
+To delete an existing role:
```

Comment:
Are there any prereqs for deleting a role? I'd think that not just anyone can delete a role.

Reply:
no, if you have admin privs, you can delete